### PR TITLE
GCW-3120 Message Center

### PR DESCRIPTION
**Pull request by swatijadhav**
### Ticket Link: 
https://jira.crossroads.org.hk/browse/GCW-3120

### What does this PR do?
Messages Center to display a list of all unread messages. 

### Screenshots
Screenshot 2020-04-16 at 3 31 21 PM

### Linked PR's:
https://github.com/crossroads/api.goodcity/pull/977

### CODE REVIEW

File: app/services/messages.js [hunk 1]
```diff
@@ -1,30 +1,90 @@
...
         .then(data => {
           delete data.message.id;
           message.setProperties(data.message);
+          this._decrementCount();
         })
         .catch(error => this.get("logger").error(error));
     }
   },
 
+  markAllRead() {
```
Comment (patrixr) [inline app/services/messages.js:45]: @swatijadhav I think this endpoint might also mark all the Admin notifications as read 🤔 , which we don't want.

But I believe this endpoint takes in a `"context"` param, which you can set to `"order"`

Comment (patrixr) [reply app/services/messages.js:45]: I guess we need to check that Admin does the same

File: app/templates/app_menu_list.hbs [hunk 2]
```diff
@@ -51,4 +51,4 @@
     </ul>
   </div>
 </div>
-{{partial "sticky_footer"}}
+{{sticky-footer}}
```
Comment (patrixr) [inline app/templates/app_menu_list.hbs:54]: 👍  good move, I think a lot of the partials are more appropriate as components around the app. 
I think we should only use partials to divide templates which are too big, not so much for making reusable pieces

File: app/services/messages.js (hunk 1 above)
Comment (swatijadhav) [reply app/services/messages.js:45]: Thanks for pointing out this one. Changes done!

**Pull request merged**